www-src/app/lib/base_view.coffee

```diff
@@ -10,9 +10,20 @@ module.exports = class BaseView extends Backbone.View
     render: ->
         @beforeRender()
         @$el.html @template(@getRenderData())
+        @bindRefs() if @refs
+        @$el.prop 'className', @className() if typeof @className is 'function'
         @afterRender()
         @
 
+    bindRefs: ->
+        for ref, selector of @refs
+            @[ref] = @$ selector
+
+    setState: (key, value) ->
+        clearTimeout @dirtyTimeout
+        @[key] = value
+        @dirtyTimeout = setTimeout @render.bind(@), 1
```

Comment:
@aenario maybe you can use a fat arrow when declaring `render` to avoid the `.bind(@)` here.